src/main/java/com/google/devtools/build/lib/starlarkbuildapi/StarlarkRuleContextApi.java

```diff
@@ -557,8 +557,8 @@ String expandLocation(
               @ParamType(type = Depset.class, generic1 = SymlinkEntryApi.class)
             },
             doc =
-                "Either a SymlinkEntry depset or the map of symlinks, prefixed by workspace name,"
-                    + " to be added to the runfiles. See <a"
+                "Either a SymlinkEntry depset or the map of symlinks to be added to the runfiles"
+                    + " with the workspace name added as a prefix. See <a"
```

Comment:
```suggestion
                "Either a SymlinkEntry depset or the map of symlinks to be added to the runfiles. Symlinks are added to the outermost, main, workspace's runfiles directory (e.g. `<runfiles_root>/_main/<symlink_path>`"
                    + " with the workspace name added as a prefix. See <a"
```

Reply:
I felt that we could be even more explicit about this behavior given how surprising it is. Could you take a look?

Reply:
I agree, LGTM.